recipes/CppInterOp/build.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
+
+set -x
+
+# Common settings
+
+export CPU_COUNT="$(nproc --all)"
+export backend=$(echo "${backend}" | tr '[:upper:]' '[:lower:]')
+export LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:${BUILD_PREFIX}/x86_64-conda-linux-gnu/lib:${BUILD_PREFIX}/lib:${PREFIX}/lib"
+#export CPLUS_INCLUDE_PATH="${CPLUS_INCLUDE_PATH}:$PWD/include:${BUILD_PREFIX}/x86_64-conda-linux-gnu/include/c++/12.3.0:${BUILD_PREFIX}/x86_64-conda-linux-gnu/include/c++/12.3.0/x86_64-conda-linux-gnu:${BUILD_PREFIX}/x86_64-conda-linux-gnu/sysroot/usr/include:${BUILD_PREFIX}/include:${PREFIX}/include"
+sys_include_path=$(LC_ALL=C x86_64-conda-linux-gnu-g++ -O3 -DNDEBUG -xc++ -E -v /dev/null 2>&1 | sed -n -e '/^.include/,${' -e '/^ \/.*++/p' -e '}' | xargs -I$ echo "$" | tr '\n' ':')
+export CPLUS_INCLUDE_PATH="${CPLUS_INCLUDE_PATH}:$PWD/include:$sys_include_path:${BUILD_PREFIX}/x86_64-conda-linux-gnu/sysroot/usr/include:${BUILD_PREFIX}/include:${PREFIX}/include"
+export CMAKE_EXTRA_CLANG_PARAMS=""
+export DO_BUILD_CLANG="0"
+
+export clangdev_tag=${clangdev/\.\*/}
+clangdev1=${clangdev_tag}.0.0
+export clangdev_ver=${clangdev1/17\.0\.0/17.0.4}  # fix: clang 17.0.0 is removed from releases
+
+# LLVM/Clang settings
+
+export LLVM_DIR="${SRC_DIR}/llvm_project"
+export LLVM_BUILD_DIR="${LLVM_DIR}/build"
+
+if [[ "${backend}" == "repl" ]]; then
+  export LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:${LLVM_BUILD_DIR}/lib:${LLVM_DIR}/lib"
+  #export CPLUS_INCLUDE_PATH="${CPLUS_INCLUDE_PATH}:${LLVM_BUILD_DIR}/tools/clang/include:${LLVM_BUILD_DIR}/include:${LLVM_DIR}/clang/include:${LLVM_DIR}/llvm/include"
+
+  x=$(conda list -q --json "^clang$")
```

Comment:
We should not fallback to building cling, clang, and LLVM if clang has not been installed. This should be considered as a critical error since a dependency that is guaranteed to be installed by conda/mamba is not.

Reply:
+ 1